lib/location.js

```diff
@@ -261,9 +275,16 @@ var LocationState = State.extend(function (self, name, opts) {
         return '\n' + number + '. ' + address.label;
     };
 
-    self.generate_footer = function() {
-        return '\n' + 'n. '+self.next_text + '\n' + 'p. '+self.previous_text;
-
+    self.generate_footer = function(content) {
+        var options = [];
+        if (self.metadata.current_page >= self.metadata.pages.length && self.retry_text){
```

Comment:
This was my initial thought as well, because otherwise your only way out is the skip option or pressing next a trillion times, but I didn't want to question the ticket